src/_includes/configure-path.html

```diff
@@ -1,21 +0,0 @@
-<div markdown="1">
```

Comment:
Do we cover setting the path anywhere?

Reply:
Whether / how one sets the path or not depends on (1) the technique used to install the SDK, (2) how the SDK is used and in which environment / shell it is used from.

For example, when using Homebrew under MacOS, one **shouldn't** adjust the path specifically for the Dart SDK. Doing so can lead to problem later on when Dart is upgraded and/or versions are switched.

Our previous instructions on how to set the path were incomplete at best, misleading in some cases.

Reply:
Let's just mention adding the SDK's bin directory to your path, and leave it up to people to find the instructions. (We added the path info because some beginners would get confused when they installed Dart but couldn't run the commands.)

Reply:
Where do you want this mentioned?

Reply:
(I'd suggest leaving any such mention for now and we can consider re-adding it, or directing folks to SO ... as the need arises. WDYT?)

Reply:
I think we should mention adding the SDK's tools to your path in the /tools/sdk page, somewhere inside the "Install the SDK" section.

Reply:
Oops, I didn't get the notice about this comment until now. I'll add it with my next PR.